docs/source/gcp/examples/pubsub-snapshots.rst

```diff
@@ -0,0 +1,26 @@
+Pub/Sub - Find Snapshots Referring to Deleted Topics
+==================================================================
+
+In Cloud Pub/Sub, the snapshot feature allows users to capture the message acknowledgment state of a subscription to a topic. Once a snapshot is created, it retains all messages that were unacknowledged in the source subscription (at the time of the snapshot's creation). Custodian can check if any of Pub/Sub snapshots refer to deleted topics (GCP uses special ``_deleted-topic_`` name for them). 
```

Comment:
please note all snapshots auto delete after at *most* a week in the docs here.. which also decreases the utility of this policy imo beyond catering to edge case/long tail optimization..

Reply:
That's right, however, per the updated description a week could still be too much in cases of regulations and someone would like to remove sensitive data sooner.

Reply:
right.. this is a what if scenario thats deep down an edge case.. not a good doc example.

Reply:
i cant think of a regulation scenario for it, and its pretty useless for most people